webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -173,8 +173,15 @@ def update_response_overview_plot(
             response = ensemble.responses[selected_response]
 
             x_axis = response.axis
+
+            if x_axis is None:
+                raise ValueError(
+                    f"No x_axis label in the response {selected_response}."
+                )
+
             if isinstance(x_axis, pd.Index) and x_axis.empty:
                 continue
```

Comment:
If i remember correctly - we can have keys which doesnt have data for certain ensembles. I guess this relates to that situation. Instead of crashing we want to just display an empty chart.